FervoEnumBundle.php

```diff
@@ -35,26 +35,18 @@ public function boot()
         $this->registerAutoloader($this->container);
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    public function shutdown()
-    {
-        if (null !== $this->autoloader) {
-            spl_autoload_unregister($this->autoloader);
-            $this->autoloader = null;
-        }
-    }
-
-    private function registerAutoloader($container)
+    private function registerAutoloader(ContainerInterface $container)
     {
-        if (!$this->autoloader) {
+        if (!$this->autoloaderRegistered) {
+            $projectDir = $container->getParameter('kernel.project_dir');
```

Comment:
`kernel.project_dir` only appeared in Symfony 3.3. We need to emulate it using `kernel.root_dir` for earlier versions.

Reply:
What do you think, should I use if statement like
```php
if (Kernel::VERSION < 30000) {
    $projectDir = $container->getParameter('kernel.root_dir').'/..';
} else {
    $projectDir = $container->getParameter('kernel.project_dir');
}
```
to simplify further search when we will drop Symfony 2.8 support or just always use `$projectDir = $container->getParameter('kernel.root_dir').'/..';`?

Reply:
No, an if statement that we can drop when 2.8 support is dropped is definitely the best.

Reply:
Updated, please review